src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -118,7 +122,7 @@ private void cancelHearing(HearingWrapper wrapper) {
     }
 
     private void partyNotified(HearingWrapper wrapper) {
-        // TODO SSCS-10075 - implement mapping for the event when a party has been notified, might not be needed
+        sendPartiesNotifiedUpdateRequest(wrapper);
```

Comment:
Why is this nested why isn't this method just in one? I know the rule is one method should have one purpose but the only purpose this has is to call another method that has one purpose?

Reply:
There should be a response here and related processing